python/triton/language/core.py

```diff
@@ -1277,7 +1278,7 @@ def swizzle2d(i, j, size_i, size_j, size_g):
 
 
 @triton.jit
-def zeros(shape, dtype):
+def zeros(shape, dtype: dtype) -> tensor:
```

Comment:
@ptillet The `dtype: dtype` annotation here may be an incorrect assumption. I see in `full` it does a potential `constexpr` conversion, so this is not necessarily a `dtype`. There may be a few others, but I believe I largely avoided this. I'll fix this one tomorrow.